src/views/shared/fiat-amount/fiat-amount.view.jsx

```diff
@@ -9,7 +9,7 @@ function FiatAmount({ amount, currency }) {
 
   return (
     <div className={classes.root}>
-      {CurrencySymbol[currency].symbol} {!isNaN(amount) ? amount.toFixed(2) : "--"}
+      {CurrencySymbol[currency].symbol} {!isNaN(amount) ? amount : "--"}
```

Comment:
This component is instantiated in `views/home/home.view.jsx` where no round is applied:

```typescript
<FiatAmount amount={totalBalanceTask.data} currency={preferredCurrency} />
```
Is it correct to remove the `.toFixed(2)` here without rounding there or somewhere else?